source/_integrations/androidtv_remote.markdown

```diff
@@ -0,0 +1,382 @@
+| `entity_id`            |       no | Entity ID to target.
+
+### Service `remote.turn_on`
+
+Turn on the Android TV. If `activity` is specified it will open the specified URL in the associated app.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            |       no | Entity ID to target.
+| `activity`             |      yes | URL to open.
+
+Examples of URLs:
+
+| App | URL |
+| --- | --- |
+| YouTube | https://www.youtube.com
+| Netflix | https://www.netflix.com/title
+| Prime Video | https://app.primevideo.com
+| Disney+ | https://www.disneyplus.com
+
+Example for launching YouTube:
+
+```yaml
+service: remote.turn_on
+data:
+  activity: https://www.youtube.com
+target:
+  entity_id: remote.living_room_tv
+```
+
+Example for opening a specific YouTube video:
+
+```yaml
+service: remote.turn_on
+data:
+  activity: https://www.youtube.com/watch?v=dQw4w9WgXcQ
+target:
+  entity_id: remote.living_room_tv
+```
+
+### Service `remote.send_command`
+
+Send a single command or a set of commands to the Android TV.
+You can optionally specify the number of times you wish to repeat the command(s), delay you want between repeated command(s), and hold.
+
+For a list of the most common commands you can send to the Android TV see: [TvKeys](https://github.com/tronikos/androidtvremote2/blob/main/TvKeys.txt).
+For a full list see [here](https://github.com/tronikos/androidtvremote2/blob/main/src/androidtvremote2/remotemessage.proto#L90).
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            |       no | Entity ID to target.
+| `command`              |       no | A single command or a list of commands to send.
+| `num_repeats`          |      yes | The number of times you want to repeat the command(s).
+| `delay_secs`           |      yes | The time you want to wait in between repeated commands.
+| `hold_secs`            |      yes | The time you want to have it held before the release is send.
+
+Example for opening the currently selected item on the Android TV:
+
+```yaml
+action: call-service
+service: remote.send_command
+data:
+  command: DPAD_CENTER
+target:
+  entity_id: remote.living_room_tv
+```
+
+Example for long pressing on the currently selected item on the Android TV:
+
+```yaml
+action: call-service
+service: remote.send_command
+data:
+  command: DPAD_CENTER
+  hold_secs: 0.5
+target:
+  entity_id: remote.living_room_tv
+```
+
+## Dashboard
+
+You have to manually create buttons in Lovelace to send commands to the Android TV device or launch apps on it.
+
+Below is an example for you to start with. Many of the buttons support long press.
+Replace all instances of `living_room_tv` with your entity ID.
+
+![Screenshot Android TV Remote example](/images/integrations/androidtv_remote/lovelace_example.png)
+
+```yaml
+type: vertical-stack
+cards:
+  - type: entities
```

Comment:
This is quite a huge piece of YAML in this documentation, and maybe a bit off-topic? Let's remove it here and feel free to create a community guide for it. A smaller or simpler example would be acceptable IMHO.

Reply:
I followed the example in https://www.home-assistant.io/integrations/xbox/#picture-elements-card which also has YAML code. Would it help if I have it initially collapsed and let users expand it?

Reply:
Sure, that works 👍

Reply:
Done.